aslam_cv_frames/include/aslam/frames/visual-frame.h

```diff
@@ -101,7 +102,14 @@ class VisualFrame  {
 
   /// Get the number of keypoints measurements stored in this frame.
   inline size_t getNumKeypointMeasurements() const {
-    return static_cast<size_t>(getKeypointMeasurements().cols());
+    size_t num_measurements = 0;
```

Comment:
Why do we modify this very basic function of aslam to include lidar keypoints? It looks super dangerous in the sense that this is actually called in quite a lot of other places including in aslam_cv which don't seem to have been adapted! While this PR here does not seem to use this function call at all itself? Should we maybe revert this particular bit and just have two separate functions `getNumKeypointMeasurements` and getNumLidarKeypointMeasurements